security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -193,7 +193,7 @@ func NewSecretCache(fetcher *secretfetcher.SecretFetcher, notifyCb func(string,
 // instead of reading from cache.
 func (sc *SecretCache) GenerateSecret(ctx context.Context, connectionID, resourceName, token string) (*model.SecretItem, error) {
 	var ns *model.SecretItem
-	key := ConnKey{
+	sdsConnCtx := ConnKey{
```

Comment:
just call connKey ?  personally i feell ctx is bit confusing.

Reply:
Done. Thanks.